tools/c7n_azure/c7n_azure/resources/network_interface.py

```diff
@@ -12,9 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from c7n_azure import constants
 from c7n_azure.provider import resources
 from c7n_azure.resources.arm import ArmResourceManager
 
+from c7n.filters.core import ValueFilter, type_schema
+from c7n_azure.utils import ThreadHelper
+import logging
+
+max_workers = constants.DEFAULT_MAX_THREAD_WORKERS
+chunk_size = constants.DEFAULT_MAX_THREAD_WORKERS

```

Comment:
Did you mean to use DEFAULT_CHUNK_SIZE ? Also if you're using the defaults you don't need to pass them since the ThreadHelper already uses the defaults.